src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -84,8 +84,11 @@
         {@processLoopView(view)}
       @case "PRINT"
         {@processPrintView(view)}
+      @case "LOOP"
```

Comment:
I think this case got merged in. It should be removed, since we handle it above.